src/pet.js

```diff
@@ -0,0 +1,70 @@
+const INITIAL_AGE = 0;
+const BASE_HUNGER = 0;
+const MAX_FITNESS = 10;
+
+function Pet(name) {
+  this.name = name;
+  this.age = INITIAL_AGE;
+  this.hunger = BASE_HUNGER;
+  this.fitness = MAX_FITNESS;
+  this.children = [];
+}
+
+Pet.prototype = {
+  get isAlive() {
+    return this.age < 30 && this.hunger < 10 && this.fitness > 0;
```

Comment:
Just as you have the BASE_HUNGER and MAX_FITNESS constants above, you could also add values for the max age and hunger and the minimum fitness levels too